mewarpx/tests/test_mwxrun.py

```diff
@@ -83,19 +85,33 @@ def test_capacitive_discharge_multigrid(capsys, name):
         mwxrun.simulation.step()
 
     #######################################################################
-    # Cleanup and final output                                            #
+    # Compare mwxrun data gathering to diagnostic output                  #
     #######################################################################
 
-    if capsys is not None:
-        out, _ = capsys.readouterr()
+    # the edges are treated differently in our gather rho versus the yt
+    # dump so we only look at the interior here
+    rho_data = np.mean(mwxrun.get_gathered_rho_grid(
+        species_name='he_ions', include_ghosts=False
+    )[0][:,1:,0], axis=0)[2:-2]
+
+    with pytest.warns(ResourceWarning):
```

Comment:
I think we should use `@pytest.mark.filterwarnings("ignore::ResourceWarning")` around the test function instead of `with pytest.warns(ResourceWarning)`, since the latter causes the test to fail if ResourceWarning is raised. A YT update that fixes the ResourceWarning raised by yt.load would break all the tests that use `with pytest.warns()`.